client/post-editor/style.scss

```diff
@@ -1,10 +1,18 @@
+/** @format */
+
+@import url( '//s1.wp.com/wp-includes/css/dashicons.css?v=20150727' );
+
+@import 'components/screen-reader-text/style';
```

Comment:
can these be moved to their respective components?

Reply:
Yes! See 794e73043b